frontend/src/metabase-lib/lib/metadata/Database.ts

```diff
@@ -73,7 +79,7 @@ export default class Database extends Base {
    * @typedef {"join"} VirtualDatabaseFeature
    * @param {DatabaseFeature | VirtualDatabaseFeature} [feature]
    */
-  hasFeature(feature) {
+  hasFeature(feature: string | undefined | null) {
```

Comment:
I'm letting `feature` be `undefined` or `null` here because of the odd block below this comment 

```
    if (!feature) {
      return true;
    }
```

🤷